app/electron/main/i18next.js

```diff
@@ -1,12 +1,17 @@
 import i18n from 'i18next';
 import i18NextBackend from 'i18next-fs-backend';
-import path from 'path';
+import {join} from 'path';
```

Comment:
should we use `node:...` format for system packages?

Reply:
I'm not sure if they will work here, since the currently used Electron version uses Node 14.16, whereas the `node:...` format was added in Node 14.18.

Reply:
node14 is so ancient :(

Reply:
That's why this PR is needed to unblock the Electron+Node upgrade :)